spring-boot/run.sh

```diff
@@ -0,0 +1 @@
+./mvnw spring-boot:run
```

Comment:
I added this run because the other platforms had it too and I had to keep checking the readme to copy/paste this mvnw command. @sdzhong